docs/source/install-tutorial.rst

```diff
@@ -115,3 +145,36 @@ Supervisord config i.e. in ``/etc/supervisor/conf.d/bepasty.conf``:
   redirect_stderr = true                                        ; Save stderr in the same log
 
 Finally reload supervisor: `service supervisor reload`
+
+In your browser go to your server address: ``http://bepasty.example.org``
+
+You should now have a running bepasty server on port 80 (HTTP) of your server.
+
+Places to look when troubleshooting:
+
+::
+
+  journalctl -u nginx.service
+  journalctl -u supervisor.service
+  tail /home/bepasty/logs/nginx-access.log
+  tail /home/bepasty/logs/nginx-error.log
+  tail /home/bepasty/logs/gunicorn_supervisor.log
+
+Important notes:
+
+* If you copied the file from the ``bepasty/config.py`` it will have
+  a "class Config" in it and all the settings are inside that class. This is
+  **not** what you need. Due to how flask config files work, you need to
+  remove the class statement and outdent all the settings, so you just have
+  global KEY = VALUE statements left on the top level of the config file.
+* When adding additional users in ``bepasty/config.py`` make sure you have
+  ``'user_secret': 'correct,list,of,permissions',``
+  any missing ``'' : '',`` will result in the bepasty server not starting.
+  
+Tips & Tricks:
+
+* The user secret in ``bepasty/config.py`` needs to be secure - an easy way 
+  to create secure random character strings on the command line is:
+  ``tr -dc A-Za-z0-9 </dev/urandom | head -c 10 ; echo ''``
+  this can also be used to create a long random secure SECRET_KEY value:
+  ``tr -dc 'A-Za-z0-9!"#$%&'\''()*+,-./:;<=>?@[\]^_`{|}~' </dev/urandom | head -c 255  ; echo ''``
```

Comment:
Maybe use same command for both and just tell to use a longer secret for the SECRET_KEY.

Reply:
The difference is that the first example only uses characters that can be typed on any language keyboard.  The second uses almost the full list of the suggested characters for strong keys - of which may be difficult to enter on different keyboards.
I'm not against the idea and can modify as per your suggestion.